components/ShowPassword/ShowPassword.js

```diff
@@ -0,0 +1,37 @@
+import Image from "next/image";
+
+export const ShowPassword = ({
+  showPasswordHandler,
+  showPassword,
+  showConfirmPassword,
+}) => {
+  return (
+    <button
+      type='button'
+      className='btnIcon btnIconShowPassword'
+      onClick={showPasswordHandler}
+    >
+      {showPassword ? (
```

Comment:
No need to use two `Image` components. Outer ternary and the ternary operator inside the `src` prop of the `Image` component are doing the same thing. Please check it once.